packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -554,13 +554,30 @@ export namespace EmrCreateCluster {
     TERMINATE_CLUSTER = 'TERMINATE_CLUSTER',
   }
 
+  /**
+   * Spot Allocation Strategies
+   *
+   */
+  export enum SpotAllocationStrategy {
+    /**\
+     * capacity-optimized
```

Comment:
Add more detail (full sentence) about what this means (can copy/paste from AWS docs)